src/utils.spec.tsx

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright 2024 Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as utils from './utils';
+import React, { forwardRef } from 'react';
+import { render, screen } from '@testing-library/react';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+import { UserInfo } from './utils';
+
+describe('utils', () => {
+  describe('areUsersEqual', () => {
+    const user = { id: '1', attributes: { name: 'user1' } };
+
+    it('returns true if users are equal', () => {
+      const user2 = JSON.parse(JSON.stringify(user));
+
+      expect(utils.areUsersEqual(user, user2)).toBe(true);
```

Comment:
NIT: Is it clearer to separate out the "act"? 

```suggestion
      const user2 = JSON.parse(JSON.stringify(user));
      
      const usersAreEqual = utils.areUsersEqual(user, user2)

      expect(usersAreEqual).toBe(true);
```

Reply:
Yes. we can do that. That would be more readable. Let me update! Thanks for the suggestion